app/src/main/java/br/com/catbag/gifreduxsample/ui/AnvilRenderComponent.java

```diff
@@ -0,0 +1,9 @@
+package br.com.catbag.gifreduxsample.ui;
+
+/**
+ * Created by raulcca on 10/31/16.
+ */
+
+public interface AnvilRenderComponent {
```

Comment:
Now they pass setListener(null) to stop listen, but with the issue opened to refactor all this piece of code, we have to put a direct removeListener(this) to remove the registered object. **See the next comment**